rules/network/command_and_control_halfbaked_beacon.toml

```
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/07/06"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Halfbaked is a malware family leveraged to establish persistence in a contested network. This rule detects a network
```

```suggestion
Halfbaked is a malware family used to establish persistence in a contested network. This rule detects a network
```
Just to avoid the double use of _leveraged_ 🙂